.evergreen/run-tests.sh

```diff
@@ -19,16 +19,22 @@ if [ "$SANITIZE" != "false" ]; then
     SANITIZE_STATEMENT="--sanitize ${SANITIZE}"
 fi
 
+# work around https://github.com/mattgallagher/CwlPreconditionTesting/issues/22 (bug still exists in version 1.x
```

Comment:
I don't run into this locally on Xcode 13.4.1 but the hosts don't have Xcode versions that new yet